sample_app/main_linux.c

```diff
@@ -1020,47 +1106,65 @@ int main(int argc, char *argv[])
    printf("\n** Starting Profinet demo application **\n");
    if (verbosity > 0)
    {
-      printf("Verbosity level: %u\n", verbosity);
+      uint8_t macbuffer[6];
+      os_cpy_mac_addr(macbuffer);
```

Comment:
The profinet stack should not get the mac address like this but rather by configuration, but we can fix that in another PR.